parts/linux/cloud-init/artifacts/azlosguard/cse_helpers_osguard.sh

```diff
@@ -41,5 +41,19 @@ tdnf_download() {
     echo Executed tdnf install -y \"$@\" $i times;
 }
 
+tdnf_update() {
```

Comment:
could we move this to  common file ? looks like it's a full duplicate of the one used in `cse_helpers_mariners.sh`